hc-dna/zomes/social_context/Cargo.toml

```diff
@@ -13,9 +13,9 @@ derive_more = "0"
 serde = "1"
 lazy_static = "*"
 chrono = { version = "0.4", features=["serde"] }
-hc_time_index = { git = "https://github.com/holochain-open-dev/holochain-time-index", branch = "main" }
+hc_time_index = { git = "https://github.com/perspect3vism/holochain-time-index", branch = "hdk-0.0.108" }
```

Comment:
Note that I'm pointing to the forked holochain-time-index. About to put another PR there..